test.sh

```diff
@@ -1,14 +1,17 @@
-#!/bin/bash
-
+#!/usr/bin/env bash
 # Run all known IncludeOS tests.
 #
 # A lot of these tests require vmrunner and a network bridge.
 # See https://github.com/includeos/vmrunner/pull/31
 
+pushd "$(dirname "$0")" >/dev/null
 : "${QUICK_SMOKE:=}" # Define this to only do a ~1-5 min. smoke test.
 : "${DRY_RUN:=}"     # Define this to expand all steps without running any
 : "${CCACHE_FLAG:=}" # Define as "--arg withCcache true" to enable ccache.
 
+UNIT_TESTS=./test/tests-unit/
+INTEGRATION_TESTS=./test/tests-integration/
```

Comment:
I don't like the name duplication - ./test/tests-integration - it's long to type and I don't see any reason for it. I prefer test/integration/